statuts_asso/statuts_osgeo.md

```diff
@@ -223,22 +260,20 @@ réunira selon les mêmes conditions d'organisation que celles de l'AGO.
 
 Les ressources de l'association se composent :
 
```

Comment:
je ne détaillerais pas trop ce paragraphe...

Reply:
> je ne détaillerais pas trop ce paragraphe...

on peut regrouper la partie subvention, dons, sponso, etc.. en une seule ligne. 
Je peux aussi enlever le revenu de nos biens.. on a pas grand chose à vendre.

Reply:
done in 70bbea85fd5706ddc08d44229775fb00048a3faa